nextflow.config

```diff
@@ -20,6 +20,10 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = false
 
+    //Gene score configuration Defaults
+    gene_score_method          = "PLAGE"
+    compute_gene_scores        = false
```

Comment:
do we need this or can we compute it as soon as the yml is provided?

Reply:
Actually can do that when something is supplied automatically yes.